### PR TITLE
docs: update onboarding documentation

### PR DESCRIPTION
**Pull request by TuringLovesDeathMetal**

### CODE REVIEW

File: docs/docs/get-started/exploring-data/sharing-insights.md [hunk 1]
```diff
@@ -13,28 +13,52 @@ In Lightdash there are three primary ways to share insights with your team:
...
-## 1. Share a URL to work in progress
+Before you do *any* sharing though, you need someone to share it with.
+
+## Inviting your teammates to your Lightdash project
+
+You can only share things with people you've invited to your Lightdash project.
+
+To invite your teammates, just click on `Settings` --> `Invite` --> `Create invite link`, then send your magic link to any of your teammates 🪄
+
+![invite user action](./assets/invite_user_action.gif)
```
Comment (owlas) [inline docs/docs/get-started/exploring-data/sharing-insights.md:24]: Should this still be a link to the gif?

Its causing an error in the build;

> 4:35:54 PM: Error: Image docs/get-started/exploring-data/assets/invite_user_action.gif used in docs/get-started/exploring-data/sharing-insights.md not found.

**Review by owlas — CHANGES_REQUESTED** [review]
There's just one broken link!

File: docs/docs/get-started/exploring-data/sharing-insights.md (hunk 1 above)
Comment (TuringLovesDeathMetal) [reply docs/docs/get-started/exploring-data/sharing-insights.md:24]: ugh, okay yes, this was me trying to be sneaky and just merge something and then add pictures later

**Comment by TuringLovesDeathMetal** [comment]
@owlas now it's saying that this file doesn't exist, even though it's in there 🤔 

docs/docs/help-and-contact/contact/assets/star_github_action.png

**Comment by owlas** [comment]
@TuringLovesDeathMetal I've updated the branch!

Could you check if anything is missing? Looks like the diff is the same to me. The error in the build is because the FAQ page is still referenced in the sidebar

**Comment by TuringLovesDeathMetal** [comment]
@owlas sorry to drag you in again...

I feel like it's so close!

I keep getting this error about the link relative paths not being correct:
image


But I'm like 99% sure that those files exist at those relative paths. Can you see if I'm missing anything here?? 

(here's the "using-explores.md" relative to the `create-metrics.md`)
image

(here's the `metrics.md` relative to the `create-metrics.md`)
image

File: docs/docs/get-started/exploring-data/sharing-insights.md [hunk 2]
```diff
@@ -13,28 +13,52 @@ In Lightdash there are three primary ways to share insights with your team:
...
-## 1. Share a URL to work in progress
+Before you do *any* sharing though, you need someone to share it with.
+
+## Inviting your teammates to your Lightdash project
+
+You can only share things with people you've invited to your Lightdash project.
+
+To invite your teammates, just click on `Settings` --> `Invite` --> `Create invite link`, then send your magic link to any of your teammates 🪄
+
+[invite user action pic]
```
Comment (owlas) [inline docs/docs/get-started/exploring-data/sharing-insights.md:24]: @TuringLovesDeathMetal do you still want this in there?

File: docs/docs/get-started/exploring-data/sharing-insights.md [hunk 3]
```diff
@@ -13,28 +13,52 @@ In Lightdash there are three primary ways to share insights with your team:
...
 
-[pic]
+![export csv action](./assets/export_csv_action.png)
 
-## 3. Download results as a table or chart
+It's also easy to download your favourite chart images. You can pick between JPEG, PNG, and more!
 
+If you're stuck for a last minute gift idea, we've heard that framed chart art can bring a smile to most people's faces 🖼
 
+[export image action pic]
```
Comment (owlas) [inline docs/docs/get-started/exploring-data/sharing-insights.md:64]: and here?

**Review by owlas — CHANGES_REQUESTED** [review]
I think you wanted these left in as placeholders @TuringLovesDeathMetal ?

File: docs/docs/get-started/exploring-data/sharing-insights.md (hunk 2 above)
Comment (TuringLovesDeathMetal) [reply docs/docs/get-started/exploring-data/sharing-insights.md:24]: yes! I just wanted to merge these changes and then figure out the images next week/on monday.

**Review by owlas — APPROVED** [review]

**Pull request merged**